yml/dp02_patches.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: dp02_dc2_images
+"@id": "#dp02_dc2_images"
+description: Information about image data products in DP0.2 processing of DC2
+tables:
+- name: lsst_skymap
```

Comment:
All the other tables are CamelCase.

Reply:
Again, I'm happy to remove the prefix, and we can make it "Skymap" if you prefer.